086-archive-canary.md

```diff
@@ -0,0 +1,92 @@
+# Archive Canary project written in Go
+
+This proposal is about archiving the [Canary project written in Go](https://github.com/strimzi/strimzi-canary).
+
+## Current situation
+
+The Canary project, written in Go programming language, is a component for monitoring a Kafka cluster.
+It provides functionality to periodically check the availability of the Kafka cluster during normal run, upgrades, downgrades, and rolling updates.
+That is done by connection check to the cluster, but also producer and consumer, that periodically do the message transmission between Kafka cluster and Canary.
+The Canary then provides Prometheus metrics and alerts for users or cluster admins to react on the Kafka cluster issues.
+
+## Motivation
+
+The Canary, as mentioned, is written in Go, however the Strimzi organization (and the engineers working on the projects inside this organization) is focused on using the Java programming language.
+This means the team is missing the necessary expertise in Go in order to solve various issues in Canary.
```

Comment:
You do not need to change it. But I think this is not the real reason and it is just an excuse. The real reason is IMHO just not enough time and interest to keep maintaining it.

Reply:
Yeah I can add it as a main point. Thanks